src/main/pages/che-7/administration-guide/proc_configuring-prod-short-to-use-external-keycloak.adoc

```diff
@@ -0,0 +1,77 @@
+// configuring-authorization
+
+[id="configuring-prod-short-to-use-external-keycloak_{context}"]
+= Configuring {prod-short} to use an external {identity-provider} installation
+
+By default, {prod-short} installation in multiuser mode includes the deployment of a dedicated {identity-provider} instance. However, using an external {identity-provider} is also possible. This option is useful when a user has an already existing {identity-provider} instance with already defined users.
+
+.Prerequisites
+
+ifeval::["{project-context}" == "che"]
+* This procedure is only applicable to {prod-short} installations done using the {prod-short} Operator. When using the `{prod-cli}` management tool and the `helm` installation mode to install {prod-short}, no supported method is available to use an external {identity-provider} instance.  
+endif::[]
+
+*  In the administration console of the {identity-provider} external installation, define a link:https://www.keycloak.org/docs/latest/server_admin/#_create-realm[realm] that will contain the users intended to connect to {prod-short}. The following steps use the example `realm-for-users` realm name.
++
+image::keycloak/external_keycloak_realm.png[link="{imagesdir}/keycloak/external_keycloak_realm.png]
+
+
+* In this `realm`, define an link:https://www.keycloak.org/docs/latest/server_admin/#oidc-clients[OIDC client] that {prod-short} will use to authenticate the users. The following steps use the example `public-client` name. Here is an example of such a client with the correct settings:
++
+image::keycloak/external_keycloak_public_client.png[link="{imagesdir}/keycloak/external_keycloak_public_client.png]
+
+
+[CAUTION]
+====
+* {prod-short} only supports `public` OIDC clients. Therefore, selecting the `openid-connect` `Client Protocol` option and the `public` Access Type option is highly recommended.
+
+* The list of `Valid Redirect URIs` must contain at least 2 URIs related to the {prod-short} server, one using the `http` protocol and the other `https`. These URIs must contain the base URL of the {prod-short} server, followed by `/*` wildcards.
+
+* The list of `Web Origins` must contain at least 2 URIs related to the Che server, one using the `http` protocol and the other `https`. These URIs must contain the base URL of the {prod-short} server, without any path after the host.
```

Comment:
same question here ... given CRW 2.3 requires TLS, is there even a way to refer to the che server route by http url?

Reply:
Well, yes, we could drop http and keep https for the default case, but having it is just a way to manage all the cases (even non-default ones).
BTW I just added in the docs what is done by the operator itself in case of internal Keycloak.

But if you have a strong opinion about that I assume we can remove the `http` one.